crates/turborepo-scm/src/git.rs

```diff
@@ -64,7 +66,38 @@ pub fn changed_files(
     Ok(files)
 }
 
+// Gets the system version of both paths by calling `fs_util::canonicalize`,
+// then strips the `monorepo_root` from the `file_path`.
+fn get_stripped_system_file_path(
+    repo_root: &ProjectRoot,
+    file_path: &Path,
+    monorepo_root: &ProjectRelativePath,
```

Comment:
Not necessary in this PR, but I find `turbo_root` to be clearer at what this param describes